### PR TITLE
Feature/sticker

### PR DESCRIPTION
**Pull request by Tougee**

### CODE REVIEW

File: app/src/main/java/one/mixin/android/job/RefreshStickerJob.kt [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package one.mixin.android.job
 
 import com.birbit.android.jobqueue.Params
+import one.mixin.android.db.insertWithCreatedAt
 import one.mixin.android.vo.Sticker
 
-class RefreshStickerJob(private val albumId: String) : BaseJob(Params(PRIORITY_UI_HIGH)
+class RefreshStickerJob(private val stickerId: String) : BaseJob(Params(PRIORITY_SEND_ATTACHMENT_MESSAGE)
```
Comment (crossle) [inline app/src/main/java/one/mixin/android/job/RefreshStickerJob.kt]: Priority use UI

File: app/src/main/java/one/mixin/android/job/DecryptMessage.kt [hunk 2]
```diff
@@ -243,12 +243,12 @@ class DecryptMessage : Injector() {
             data.category.endsWith("_STICKER") -> {
                 val decoded = Base64.decode(plainText)
                 val mediaData = GsonHelper.customGson.fromJson(String(decoded), TransferStickerData::class.java)
-                val sticker = stickerDao.getStickerByUnique(mediaData.albumId, mediaData.name)
+                val sticker = stickerDao.getStickerByUnique(mediaData.stickerId)
```
Comment (crossle) [inline app/src/main/java/one/mixin/android/job/DecryptMessage.kt]: Check stickerId or name and albumId

File: app/src/main/java/one/mixin/android/job/RefreshAddressJob.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    companion object {
+        private const val serialVersionUID = 1L
+        const val GROUP = "RefreshAddressJob"
+    }
+
+    override fun onRun() {
+        val response = addressService.address(addressId).execute().body()
+        if (response != null && response.isSuccess && response.data != null) {
+            response.data.let {
```
Comment (crossle) [inline app/src/main/java/one/mixin/android/job/RefreshAddressJob.kt]: response.data?.let {}

**Comment by Tougee** [comment]
All fixed.

**Pull request closed without merging**